content/v2.0-preview/composition/composite-resources.md

```diff
@@ -21,41 +21,63 @@ Crossplane has four core components that users commonly mix up:
   (`XRD`) - A custom API specification. 
 * Composite Resource (`XR`) - This page. Created by
   using the custom API defined in a Composite Resource Definition. XRs use the
-  Composition template to create new managed resources. 
+  Composition template to create new resources. 
 {{</expand >}}
 
-## Creating composite resources
+## Create composite resources
 
 Creating composite resources requires a 
 [Composition]({{<ref "./compositions">}}) and a 
 [CompositeResourceDefinition]({{<ref "./composite-resource-definitions">}}) 
 (`XRD`).  
-The Composition defines the set of resources to create.  
-The XRD defines the custom API users call to request the set of resources.
 
-![Diagram of the relationship of Crossplane components](/media/composition-how-it-works.svg)
+The Composition defines the set of resources to create. The XRD defines the
+custom API users call to request the set of resources.
+
+```mermaid
+flowchart TD
+
+user(["User"])
+xr("Composite Resource (XR)")
+xrd("Composite Resource Definition (XRD)")
```

Comment:
there seems to be some padding/clipping issues in the mermaid diagrams for longer strings. See how `Composite Resource Defin` is clipped here:

<img width="246" alt="Screenshot 2025-03-31 at 7 26 28 AM" src="https://github.com/user-attachments/assets/3a042ce3-8b8f-4df8-8d6a-e95e1c57451f" />

Reply:
Interesting - not an issue for me in Safari.

Reply:
🤔 I asked a few folks and I'm told this renders correctly on Safari, Chrome, Firefox, and Edge. Just gonna leave it for now.